app/controllers/foreman_bootdisk/api/v2/disks_controller.rb

```diff
@@ -10,7 +10,6 @@ class DisksController < ::Api::V2::BaseController
           api_base_url "/bootdisk/api"
         end
 
-        before_filter :find_resource, :only => :host
```

Comment:
I agree in general, but this particular helper is actually misleading, we search for host in disks controller. The code is cleaner without the helper actually in my opinion. In addition, we will be moving subnet image button to subnets page, therefore the actions will likely move somewhere else.